src/ent/static_eval.h

```diff
@@ -176,10 +175,10 @@ static inline int static_eval_get_move_score(const LetterDistribution *ld,
     col_start = tmp_start;
   }
 
-  int ls;
-  int main_word_score = 0;
-  int cross_scores = 0;
-  int word_bingo_bonus = 0;
+  Equity ls;
+  Equity main_word_score = 0;
```

Comment:
Not sure what the difference between setting to 0 and setting to EQUITY_ZERO_VALUE is here, but it seems intentional, so just let me know the logic behind it.